master/master.cfg

```diff
@@ -550,7 +550,9 @@ perf_factory.addStep(ShellCommand(
     command=["runurl", bb_url + "bb-test-cleanup.sh"],
     haltOnFailure=False, maxTime=120, sigtermTime=30, logEnviron=False,
     lazylogfiles=True,
-    logfiles={"test"     : { "filename" : "TEST",        "follow" : False }},
+    logfiles={"test"            : { "filename" : "TEST",                   "follow" : False },
+              "coverage-user"   : { "filename" : "coverage-user.tar.xz",   "follow" : True  },
+              "coverage-kernel" : { "filename" : "coverage-kernel.tar.xz", "follow" : True  }},
```

Comment:
For the tarballs you're going to want to set `"follow" : False, "lazylogfiles" : True`, the options are describe [here](http://docs.buildbot.net/0.8.1/Using-ShellCommands.html).  This way the links to the logs should only appear for the coverage builder.

You should also add this to the `test_factory` and not the `perf_factory`.